src/server/images/controllers.js

```diff
@@ -29,16 +29,7 @@ export default {
         }));
       }
       Promise.all(promises).then(() => {
-        return new Promise((resolve, reject) => {
-          const conn = new S3Connector();
-          conn.getImageUrls(keys, (err, urls) => {
-            if (err) {
-              reject();
-              return;
-            }
-            resolve(urls);
-          });
-        });
+        return new S3Connector().getImageUrls(keys);
```

Comment:
I think we can remove this 'then' block and just use 
urls = new S3Connector().getImgaeUrls(keys)